orttraining/orttraining/python/training/optim/lr_scheduler.py

```diff
@@ -0,0 +1,290 @@
+import math
+
+from . import config
+
+
+class LRScheduler(object):
```

Comment:
I am assuming that we expect users to inherit from this base class which is why it is not named with a leading `_` (internal). Is that correct?

Reply:
correct